ci-operator/jobs/openstack-k8s-operators/openstack-operator/openstack-k8s-operators-openstack-operator-master-presubmits.yaml

```diff
@@ -309,6 +309,70 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )openstack-operator-build-deploy,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
Should we consider moving to "always_run: false"? since we already have the build_deploy one always running in this repo? Or maybe the reverse, set the previous build&deploy job to false.

Reply:
Done